fontforge/savefont.c

```diff
@@ -84,9 +82,7 @@ const char *savefont_extensions[] = { ".pfa", ".pfb", ".bin", "%s.pfb", ".pfa",
 	".ufo2",
 	".ufo3",
 	".woff",
-#ifdef FONTFORGE_CAN_USE_WOFF2
 	".woff2",
-#endif
```

Comment:
I'm not thrilled about this. Seems better to
```
#ifdef FONTFORGE_CAN_USE_WOFF2	
	".woff2",
#else
	NULL,
#endif
```
No harm in an extra NULL to preserve the length.

Reply:
Not sure if it gains us much. With my change `font.generate("font.woff2")` does nothing, with your proposed change it generates a *Type 1* font with the `font.woff2` extension

Reply:
In the GUI, the WOFF2 entry is greyed either way.